client/landing/subscriptions/index.tsx

```diff
@@ -0,0 +1,19 @@
+import 'calypso/components/environment-badge/style.scss';
+import '@automattic/calypso-polyfills';
+import { Reader } from '@automattic/data-stores';
+import { SubscriptionManagerApp } from '@automattic/subscription-manager';
+import ReactDom from 'react-dom';
+import { initializeCurrentUser } from 'calypso/lib/user/shared-utils';
+
+declare const window: Window & {
+	AppBoot?: () => void;
+};
+
+window.AppBoot = async () => {
+	const user = await initializeCurrentUser();
```

Comment:
Are we planning to use the bootstrapped user here on production?

Reply:
Yes, but only for checking whether the current user is a logged-in user. 

However, I think that ideally we would like to have a check in our express server before we decide to load the entry point. Then, in case we are dealing with a logged-in user, we would redirect them to the reader part of calypso. This subscription-manager portal is meant only for external (not logged-in) users.

Reply:
However, I would like to address this in a follow-up PR, if you do not mind. There is ongoing work on the subscription manager parallel to my PR. The sooner we merge it, the fewer conflicts will be caused. I will create a task on our board for every issue that we discuss on this PR, and that is not completed as part of this PR. I hope you are OK with this @tyxla . 🙂

Reply:
I created the following task for the above: https://github.com/Automattic/wp-calypso/issues/74886